### DISCUSSION TITLE
Seeking for help for disabling Youtube shortcuts keys

### DISCUSSION THREAD

**Discussion by kkl1919** [uBlockOrigin/uAssets#19869] [General]
Hi, I encountered a problem here. I wish to disable all the Youtube shortcuts key but not the 'f' and 't' key, I pasted ' youtube.com##+js(addEventListener-defuser.js, keydown) ' into uBlockOrigin dashboard but what it did is to disable all the keys, but I wish to keep the 'f' and 't' keys, do anyone know what should I type in the dashboard instead? Thanks!

**Comment by krystian3w** [comment]
In uBo dev you can try log all readable events with keyboard to devtools console:

https://github.com/gorhill/uBlock/wiki/Resources-Library#addeventlistener-defuserjs-

See "log" section in "tokens" for "scriptlet".


---

If Google doesn't mask events from uBo, this may work:

```adb
youtube.com##+js(addEventListener-defuser.js, keydown, /^(?!.*(70|84)).*/)
``` 
or maybe
```adb
youtube.com##+js(addEventListener-defuser.js, keydown, /^(?!.*key(F|T)).*/)
```

Events and "negative lookahead" are not my passion, so I won't be much help.
